models/Event.js

```diff
@@ -334,7 +343,34 @@ const Event = sequelize.define('event', {
         type: Sequelize.TEXT,
         allowNull: true
     },
-}, {
+    optional_programme: {
+        type: Sequelize.TEXT,
+        allowNull: true
+    },
+    link_info_travel_country: { // not validated. Will be checked client side, but validated nowhere
```

Comment:
not sure what does this field means, what it'll be used for?

Reply:
From the ticket; "Please insert a link which provides useful information about travelling to your country! (optional open question, string)"